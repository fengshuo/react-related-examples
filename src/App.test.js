import React from 'react';
import ReactDOM from 'react-dom';
import CommentBox from './App';
import renderer from 'react-test-renderer';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
// });
jest.mock('react/lib/ReactDefaultInjection');

it('renders correctly', () => {
  const tree = renderer.create(
    <CommentBox url="http://localhost:3001/api/comments"></CommentBox>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
