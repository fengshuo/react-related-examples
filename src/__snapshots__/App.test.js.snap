exports[`test renders correctly 1`] = `
<div
  className="commentBox">
  <div
    className="commentList" />
  <form
    className="commentForm"
    onSubmit={[Function bound handleSubmit]}>
    <input
      onChange={[Function bound handleAuthorChange]}
      placeholder="your name"
      type="text"
      value="" />
    <input
      onChange={[Function bound handleTextChange]}
      placeholder="your content"
      type="text"
      valur="" />
    <input
      type="submit"
      value="Post Your Comment" />
  </form>
</div>
`;
